pkgpanda/build/src_fetchers.py

```diff
@@ -112,7 +112,10 @@ def checkout_to(self, directory):
                                                   origin_commit))
 
         # Clone into `src/`.
-        check_call(["git", "clone", "-q", self.bare_folder, directory])
+        if is_windows:
+            check_call(["git", "clone", "-q", "--config", "core.autocrlf=true", self.bare_folder, directory])
+        else:
+            check_call(["git", "clone", "-q", self.bare_folder, directory])
```

Comment:
This could use a comment explaining why it's a different command on Windows.

Reply:
This actually brings up an interesting question. This actually is needed for Mesos specifically and I believe nothing else. Platform crlf is complicated and is different between go and other projects. How can we put some configuration on a package to turn on this functionality as it is not going to be needed for everything.

Reply:
@branden thoughts? For the initial check-in I think we can leave this as is.

Reply:
I'd need more context to understand this. Why does (only) Mesos require autocrlf to build?